notifications.bs

```diff
@@ -258,8 +259,10 @@ important information through, e.g., loss of color or clipped corners.
  <li><p>Set <var>notification</var>'s <a for=notification>actions</a> to « ».
 
  <li>
-  <p>For each <var>entry</var> in <var>options</var>["{{NotificationOptions/actions}}"], up to the
-  <a>maximum number of actions</a> supported (skip any excess entries):
+  <p>Let |actions| be a [=list=] of initial maximum two items from
```

Comment:
This is my best effort to make a sublist without actual corresponding infra term. Filed https://github.com/whatwg/infra/issues/666.

I wonder we should throw here, though.